templates/macros/form.njk

```diff
@@ -22,6 +22,9 @@
     {% if hint %}
       <span class="hint">{{hint}}</span>
     {% endif %}
+    {% if linkHint %}
+      <span class="hint">{{linkHint}} {% if link %}<a href="{{link}}" target="_blank">{{link}}</a>{% endif %}</span>
```

Comment:
We should add the attribute `rel="noopener noreferrer"` whenever we are doing a link with `target=_blank` (see https://support.performancefoundry.com/article/186-noopener-noreferrer-on-my-links for more info) - but I've seen there is another PR open (https://github.com/VLCTechHub/VLCTechHub-site/pull/73) that seems to be based on this code and adds that, so I'm approving this PR. Thanks a lot!

Reply:
Thanks for the info! I'll use it next time 👍